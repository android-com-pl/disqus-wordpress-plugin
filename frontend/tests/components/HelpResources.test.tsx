import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';
import HelpResources from '../../src/ts/components/HelpResources';

test('SupportLinks renders with correct links', () => {
    const component = TestRenderer.create(<HelpResources />);

    expect(component).toMatchSnapshot();
});
