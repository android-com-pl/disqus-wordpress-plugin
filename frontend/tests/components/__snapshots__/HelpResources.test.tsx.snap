// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SupportLinks renders with correct links 1`] = `
<ul>
  <li>
    <a
      href="https://status.disqus.com/"
      target="_blank"
    >
      Disqus System Status
    </a>
  </li>
  <li>
    <a
      href="https://github.com/disqus/disqus-wordpress-plugin"
      target="_blank"
    >
      Github Project
       (disqus-wordpress-plugin)
    </a>
  </li>
</ul>
`;
